ui/src/app/components/cases/test-case-details.component.html

```diff
@@ -40,7 +40,7 @@
             *ngIf="!testCase?.deleted && false"
             [versionId]="testCase?.workspaceVersionId"
             [testCase]="testCase"></app-test-case-detail-pagination>
-          <div *ngIf="version?.workspace?.isWeb && !testCase?.deleted">
+          <div *ngIf="(version?.workspace?.isWeb || version.workspace.isMobileWeb) && !testCase?.deleted">
```

Comment:
@nixonakash01 Have you tested whether the recording works fine in mobile web ?

Reply:
checked it. it works in mobile web
![image](https://user-images.githubusercontent.com/106724526/225223677-48d33f8b-e9d3-4ff4-9bcc-4e44aa47c2e2.png)

Reply:
@nixonakash01 The screenshot doesn't help. Check that the recording happens properly once you click on record button in mobile web

Reply:
@tarun-testsigma steps are recorded, i checked it

Reply:
Ok